src/builtin_unset.c

```diff
@@ -0,0 +1,25 @@
+#include "bash_ops.h"
+#include "env_singleton.h"
+#include "ft_printf.h"
+
+int	builtin_unset(t_command *cmd)
+{
+	int	count;
+	int	exit_status;
+
+	if (cmd->argv->argc < 2)
+	{
+		/*later use error handler ?*/
+		ft_dprintf(STDERR, "unset: not enough arguments");
+		return (-1);
+	}
+	count = 1;
```

Comment:
wouldn't index make a lot more sense since this is not a length or size, but index ?